browser/js/home/home.html

```diff
@@ -1,4 +1,16 @@
 <section id="home">
-    <fullstack-logo></fullstack-logo>
-    <h1><a ui-sref="films">All Films</a></h1>
-</section>
+
+<h1>Top Films</h2>    
+
+    <div class="film-area">
+  <div class="film-list col-xs-6 col-md-3 col-lg-3" ng-repeat="film in films">
+  	<a ng-href="/products/{{film._id}}">
+      <img ng-src="{{film.photo}}" />
+      <p id="movie-title">{{film.title}}</p>
+    </a>
+    <span id="movie-price">{{film.price | centsToDollars | currency}}</span>
+   <button type="button" class="btn btn-success">BUY</button>
```

Comment:
you can copy the code that got merged from the all films page to hook this button up to real 'buy'ing! :grinning: